pytorch_tools/torchfold.py

```diff
@@ -33,14 +34,56 @@ def nobatch(self):
 
         def get(self, values):
             if self.split_idx >= 0:
-                return values[self.step][self.op][self.split_idx][self.index]
+                return values[self.step][self.op].get(self.index, self.split_idx)
```

Comment:
can just here call `values[self.step][self.op].get(self.index, self.split_idx)` without if, right?

Reply:
Yes, I missed that.